spec/decrypt_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'aes256gcm_decrypt'
+require 'base64'
+require 'digest'
+
+describe 'Aes256GcmDecrypt::decrypt' do
+  before(:each) do
+    @ciphertext_and_tag = Base64.decode64(File.read('spec/token_data_base64.txt'))
+    @key = [File.read('spec/key_hex.txt').strip].pack('H*')
+  end
+
+  it 'decrypts correctly' do
+    plaintext = Aes256GcmDecrypt::decrypt(@ciphertext_and_tag, @key)
+    digest = Digest::SHA256.base64digest(plaintext)
+    expect(digest).to eq '6Nltyg0FgJxJ8wt6D6XDZ0y4iRD53kLoQcTBQ0FEvyY='
```

Comment:
Why test for the SHA digest rather than "just" the decrypted string? We could do without `require 'digest'` which I wondered a bit about 🙂

Reply:
I think you are right. Comparing to explicit plaintext is simpler to grasp.

:+1: